rpcserver.go

```diff
@@ -386,6 +386,20 @@ func (r *rpcServer) WalletBalance(ctx context.Context,
 	return &lnrpc.WalletBalanceResponse{balance.ToBTC()}, nil
 }
 
+
+func (r *rpcServer) ChannelBalance(ctx context.Context,
+	in *lnrpc.ChannelBalanceRequest) (*lnrpc.ChannelBalanceResponse, error) {
+
+	var balance btcutil.Amount
+	for _, peer := range r.server.Peers() {
+		for _, snapshot := range peer.ChannelSnapshots() {
+			balance += snapshot.Capacity
```

Comment:
As mentioend above, the `ChannelBalance` command should sum of the _available_ channel balance instead of the _total_ capacity. So this should instead sum up all the `snapshot.LocalBalance`'s,